board/app/controllers/user_controller.php

```diff
@@ -1,29 +1,30 @@
 <?php
 
-class UserController extends AppController{
+class UserController extends AppController {
+
+const MAX_DATA_PER_PAGE = 5;
```

Comment:
Done po :bow:
Defined it in core.php po. Thank you :blush: 
https://github.com/maragatchalian/FoodFacts/commit/ba61dd2e8f10ca1a6af2130ac72e8a8341b6a2ec